scripts/symbolize.py

```diff
@@ -16,7 +16,7 @@
 # This gets the address from lines looking like this:
 # E/TC:0  0x001044a8
 STACK_ADDR_RE = re.compile(
-    r'[UEIDFM]/T[AC]:(\?|[0-9]+) [0-9]* +(?P<addr>0x[0-9a-f]+)')
+    r'[UEIDFM]/T[AC]:(\?|[0-9]+) +[0-9]* +(?P<addr>0x[0-9a-f]+)')
```

Comment:
Actually its space before "?" as follows:

`E/TC: ? 0 Call stack:`

Reply:
Ah OK, thanks. In this case I'll add a `<space>+` and still keep the `+` that I have added for robustness.